src/lab6.py

```diff
@@ -249,6 +249,14 @@ def paint(self):
                                          word, font, color))
         return cmds
 
+    @wbetools.js_hide
```

Comment:
These methods were removed in this chapter due to a refactor; this seems a reasonable way to get rid of them from the outline.

Reply:
Yeah it would be nice to have something nice but this is definitely a good fix for now.